CodeEdit/Features/ActivityViewer/Tasks/SchemeDropDownView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SchemeDropDownView.swift
+//  CodeEdit
+//
+//  Created by Tommy Ludwig on 24.06.24.
+//
+
+import SwiftUI
+
+struct SchemeDropDownView: View {
+    @Environment(\.colorScheme)
+    private var colorScheme
+
+    @State var isSchemePopOverPresented: Bool = false
+    @State private var isHoveringScheme: Bool = false
+
+    @ObservedObject var workspaceSettingsManager: CEWorkspaceSettingsManager
+    var workspaceFileManager: CEWorkspaceFileManager?
+
+    var body: some View {
+        HStack(spacing: 5) {
+            Image(systemName: "folder.badge.gearshape")
+                .imageScale(.medium)
+
+            Group {
+                if workspaceSettingsManager.settings.project.projectName.isEmpty {
+                    Text(workspaceFileManager?.workspaceItem.fileName() ?? "No Project found")
+                } else {
+                    Text(workspaceSettingsManager.settings.project.projectName)
+                }
+            }.font(.subheadline)
+        }
+        .font(.caption)
+        .padding(.trailing, 9)
+        .padding(5)
+        .background {
+            Color(nsColor: colorScheme == .dark ? .white : .black)
+                .opacity(isHoveringScheme || isSchemePopOverPresented ? 0.05 : 0)
+                .clipShape(RoundedRectangle(cornerSize: CGSize(width: 4, height: 4)))
+            HStack {
+                Spacer()
+                if isHoveringScheme || isSchemePopOverPresented {
+                    chevronDown
+                        .padding(.trailing, 3)
+                } else {
+                    chevron
+                        .padding(.trailing, 3)
+                }
+            }
+        }
+        .onHover(perform: { hovering in
+            self.isHoveringScheme = hovering
+        })
+        .popover(isPresented: $isSchemePopOverPresented, arrowEdge: .bottom) {
+            VStack(alignment: .leading, spacing: 0) {
+                WorkspaceMenuItemView(
+                    workspaceFileManager: workspaceFileManager,
+                    item: workspaceFileManager?.workspaceItem
+                )
+
+                Divider()
+                    .padding(.vertical, 5)
+                Group {
+                    OptionMenuItemView(label: "Add Folder..") {
+                        // TODO: Implment Add Folder
+                        print("NOT IMPLMENTED")
+                    }
+                    OptionMenuItemView(label: "Workspace Settings...") {
+                        NSApp.sendAction(
+                            #selector(CodeEditWindowController.openWorkspaceSettings(_:)), to: nil, from: nil
+                        )
+                    }
+                }
+            }
+            .padding(5)
+            .frame(width: 215)
```

Comment:
This should be removed, otherwise longer workspace names are cut off or forced to wrap.